edx_django_utils/cache/utils.py

```diff
@@ -268,7 +268,7 @@ def _get_and_set_force_cache_miss(request):
         DEFAULT_REQUEST_CACHE.set(SHOULD_FORCE_CACHE_MISS_KEY, force_cache_miss)
 
     @classmethod
-    def _should_force_django_cache_miss(cls):
+    def _should_force_cache_miss(cls):
```

Comment:
I don't think the name of this method should change.  Note: a tiered cache has a Request Cache and Django Cache, and the miss only occurs at the Django Cache layer, and I think the name makes it a bit more clear.

Reply:
Yes I was also considering about the logic you just explained, 
As method is making decision on the bases of `SHOULD_FORCE_CACHE_MISS_KEY`, So I changed name accordingly.
And Yes, your explanation also making sense equally. I am going to revert this change.